src/libs/OptionsListUtils.ts

```diff
@@ -1833,7 +1833,8 @@ function getOptions(
     // - All archived reports should remain at the bottom
     const orderedReportOptions = lodashSortBy(filteredReportOptions, (option) => {
         const report = option.item;
-        if (option.isArchivedRoom) {
+        const reportNameValuePairs = ReportUtils.getReportNameValuePairs(option?.reportID);
+        if (ReportUtils.isArchivedRoom(option, reportNameValuePairs)) {
```

Comment:
Same thing here I believe?

Reply:
There's a couple other places where the same thing occurs. I left a comment [here](https://github.com/Expensify/App/pull/46934#pullrequestreview-2301889226)